docs/decisions/0009-dotnet-project-structure.md

```diff
@@ -208,7 +208,7 @@ SK-dotnet
 | Planner.ActionPlanner               | Semantic Kernel implementation of an action planner |
 | Planner.SequentialPlanner           | Semantic Kernel implementation of a sequential planner |
 | Planner.StepwisePlanner             | Semantic Kernel implementation of a stepwise planner |
-| TemplateEngine.PromptTemplateEngine | Prompt template engine implementation which is used by Semantic Functions only |
+| TemplateEngine.Prompt | Prompt template engine implementation which is used by Semantic Functions only |
```

Comment:
Following the pattern used by Reliability.Basic I suggest using the same approach for TemplateEngine

```suggestion
| TemplateEngine.Basic | Prompt template engine basic implementation which is used by Semantic Kernel by default |
```